README.md

```diff
@@ -2,73 +2,250 @@
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
-<!-- You can install de TOC plugin on VSCode and other text editors -->
+![docs-searchbar.js example](/assets/docs-searchbar-example.png)
 
+If you use Vuepress for your documentation, you might check out our [Vuepress plugin for MeiliSearch](https://github.com/meilisearch/vuepress-plugin-meilisearch).
+
+## Table of Contents <!-- omit in toc -->
+
+- [Installation and Usage](#installation-and-usage)
+  - [Before using the library](#before-using-the-library)
+  - [Basic Usage](#basic-usage)
+  - [Customization](#customization)
 - [Development Workflow](#development-workflow)
-  - [Requirements](#requirements)
-  - [Install](#install)
-  - [Build](#build)
-  - [Serve](#serve)
-  - [Test](#test)
-  - [Release](#release)
-- [Contributing](#contributing)
 - [Credits](#credits)
 
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+## Installation and Usage
+
+### Before using the library
+
+The goal of this library is to provide very easy integration of a search bar into your own documentation. To make that possible, you need to scrape your website content in advance, and index it in a MeiliSearch instance.
+
+Luckily, we provide all the tools that you need, and can help you through the whole process, if you follow [this guide](https://docs.meilisearch.com/resources/howtos/search_bar_for_docs.html) 🚀
+
+As a first introduction, you might only want to test this library without connecting it to your website.<br>
+You can do it by running the playground provided in this repository:
+
+```bash
+$ yarn install
+$ yarn serve
+```
+
+The data comes from MeiliSearch documentation.<br>
+Type `create an indxe` to live the MeiliSearch experience with the [typo tolerance](https://docs.meilisearch.com/guides/advanced_guides/typotolerance.html).
+
+### Basic Usage
+
+```html
+<!DOCTYPE html>
+<html>
+  <head>
+    <link
+      rel="stylesheet"
+      href="https://cdn.jsdelivr.net/npm/docs-searchbar.js@{version}/dist/cdn/docs-searchbar.min.css"
+    />
+  </head>
+
+  <body>
+    <input type="search" id="search-bar-input" />
+    <script src="https://cdn.jsdelivr.net/npm/docs-searchbar.js@{version}/dist/cdn/docs-searchbar.min.js"></script>
+    <script>
+      docsSearchBar({
+        hostUrl: "https://mymeilisearch.com",
+        apiKey: "XXX",
+        indexUid: "docs",
+        inputSelector: "#search-bar-input",
+        debug: true // Set debug to true if you want to inspect the dropdown
+      });
+    </script>
+  </body>
+</html>
+```
+
+The `hostUrl` and the `apiKey` fields are the credentials of your MeiliSearch instance.<br>
+`indexUid` is the index identifier into the scraper have pushed your website contents.<br>
+`inputSelector` is the CSS identifier of the HTML search input.
```

Comment:
I have changed `identifier` to `id` as it is the attribute's name.
I have removed the CSS part as it is not a CSS attribute but a HTML attribute that **can** be used by the CSS to give style to a given ID. 
https://www.w3schools.com/tags/att_id.asp
```suggestion
`inputSelector` is the `id` attribute of the HTML search input tag.
```